### ISSUE TITLE
[1.7.10] [1.4.6.1348] Real Train Mod

### ISSUE THREAD

**Issue by Hose313** [ForgeEssentials/ForgeEssentials#2303]
I plan on starting a Real Train Mod server and offcourse with the use of ForgeEssentials.
Ive installed them on a server without a problem. But as soon as i plant a train on the track the server crashes. 
Without ForgeEssentials there is no problem

https://pastebin.com/r9WGkAQ3

**Comment by spacebuilder2020** [comment]
It looks like it is trying to load a client side class on the server.  I will try to investigate it later when I get a chance.  To make things easier, can you post a link to the client and server zips? (removing personal information of course).

**Comment by Zethalion** [comment]
As we moved to 1.12.2, is this still an issue? Closing this issue in 7 days if no response @Hose313

**Comment by TekuSP** [comment]
Yes it's still issue. Having same problem. Three years after initial bug report...

**Comment by TekuSP** [comment]
And comments of type "We support only 1.12.2 or newer" is not really applicable here. Issue is from 2017, it's still open.

I am open to provide anything you need to trace that issue, but this is major issue, as ForgeEssentials is only version of Essentials applicable to running on Forge servers. I tried tracing issue myself, without any luck. I tried compiling from latest git 1.7.10 branch here, no luck either. I tried using latest jar available, no luck.

I don't know why Essentials causes Real Train Mod to freak out and load Client side on Server.
[crash-2021-04-13_19.46.16-server.txt](https://github.com/ForgeEssentials/ForgeEssentials/files/6312860/crash-2021-04-13_19.46.16-server.txt)

Reproducing is simple, add 1.7.10 forge server, mix into it:
RTM{1.7.10.41} [RealTrainMod] (RTM1.7.10.41_Forge10.13.4.1558.jar) 
NGTLib{1.7.10.32} [NGTLib] (NGTLib1.7.10.32_Forge10.13.4.1558.jar) 
and latest ForgeEssentials for 1.7.10, does not matter version, as I said even GIT builded failed.

**Comment by spacebuilder2020** [comment]
Development for Forge Essentials has halted due to lack of active developers.  You are welcome however to make a Pull request to 1.7.10 if you find a fix.
